client/src/components/SearchBox/SearchBox.component.jsx

```diff
@@ -97,6 +97,7 @@ const SearchBox = ({ placeholder, value, handleSubmit }) => {
             <div className="search-autocomplete">
               <InputGroup className="search-box">
                 <InputLeftElement
+                  zIndex="0"
```

Comment:
By default, the zIndex of `InputLeftElement` is 2 and will clip the menu dropdown, even if portalled.